0pdd.rb

```diff
@@ -204,8 +204,8 @@
 
 get '/snapshot' do
   content_type 'text/xml'
-  name = repo_name(params[:name])
-  repo = repo(name)
+  @name = repo_name(params[:name])
```

Comment:
@tedtoer this is a very anti-OOP approach. Don't do that. Object attributes are __not__ temporary storage for data.